packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.routing.js

```diff
@@ -1,14 +1,24 @@
 export default /* @ngInject */ ($stateProvider) => {
-  $stateProvider.state('network-security.traffic', {
-    url: '/traffic',
-    component: 'traffic',
-    params: {
-      subnet: null,
-    },
-    resolve: {
-      breadcrumb: /* @ngInject */ ($translate) =>
-        $translate.instant('network_security_dashboard_breadcrumb'),
-      getSubnet: /* @ngInject */ ($state) => () => $state.params.subnet,
-    },
-  });
+  $stateProvider
+    .state('network-security.traffic', {
+      url: '/traffic',
+      component: 'traffic',
+      params: {
+        subnet: null,
+      },
+      resolve: {
+        breadcrumb: /* @ngInject */ ($translate) =>
+          $translate.instant('network_security_dashboard_breadcrumb'),
+        getSubnet: /* @ngInject */ ($state) => () => $state.params.subnet,
+      },
+    })
+    .state('network-security.traffic.ip', {
+      url: '?ip&dateTime',
+      component: 'traffic',
+      resolve: {
+        ip: /* @ngInject */ ($transition$) => $transition$.params().ip,
+        dateTime: /* @ngInject */ ($transition$) =>
+          $transition$.params().dateTime,
+      },
+    });
```

Comment:
You're using the same controller and template.Please use the existing state with optional state params.

Reply:
I removed this resolve to keep the state params which work well.